rfc/README.md

```diff
@@ -71,3 +71,4 @@ The list of all RFCs can be found here.
 | 45 | [Asynchronous Metadata Indexing](./rfc-45/rfc-45.md) | `UNDER REVIEW` |
 | 46 | [Optimizing Record Payload Handling](./rfc-46/rfc-46.md) | `UNDER REVIEW` |
 | 47 | [Add Call Produce Command for Spark SQL](./rfc-47/rfc-47.md) | `UNDER REVIEW` |
+| 48 | [Optimized storage layout for Cloud object stores](./rfc-48/rfc-48.md) | `UNDER REVIEW` |
```

Comment:
@umehrot2 May be call this "Federated storage layer for Hudi" ... files can be across cloud storage or even on-prem and cloud